test/integration/scheduled_stop_test.go

```diff
@@ -69,8 +69,6 @@ func TestScheduledStopWindows(t *testing.T) {
 	time.Sleep(5 * time.Second)
 	// make sure minikube status is "Stopped"
 	ensureMinikubeStatus(ctx, t, profile, "Host", state.Stopped.String())
-	// make sure minikube timtostop is "Nonexistent"
-	ensureMinikubeStatus(ctx, t, profile, "TimeToStop", "Nonexistent")
```

Comment:
@medyagh, what I am thinking is instead of removing this check from the test. Can we check for `TimeToStop` status value to be `""`?

Reply:
This should check that the `timeToStop` line is omitted entirely.